extensions/amp-3q-player/0.1/amp-3q-player.js

```diff
@@ -240,6 +240,21 @@ class Amp3QPlayer extends AMP.BaseElement {
   }
 
   /** @override */
+  getMetaData() {
+    return {
```

Comment:
let's allow these to be just 
```
getMetadata() {
 /// Not implemented
}
```

and handle undefined/null in the video manager